catalog-spec/catalog-spec.md

```diff
@@ -215,7 +215,7 @@ The following types are commonly used as `rel` types in the Link Object of a STA
 
 | Type    | Description |
 | ------- | ----------- |
-| self    | **REQUIRED.** _Absolute_ URL to the catalog file itself. This is required, to represent the location that the file can be found online. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from. |
+| self    | STRONGLY RECOMMENDED. _Absolute_ URL to the location that the catalog file can be found online, if available. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from. |
 | root    | URL to the root STAC Catalog or [Collection](../collection-spec/README.md). Catalogs should include a link to their root, even if it's the root and points to itself. |
```

Comment:
Can/should we strongly recommend this one too?

Reply:
Probably. I can tackle that in my PR on best practices, as I'll likely hit the core a bit more too. I think 'root' is a bit vague in the current writing, so can flesh it out some more.

Reply:
Will do.